### ISSUE TITLE
Alias Server Support

### ISSUE THREAD

**Issue by sinasalek** [Real-Gecko/virtualmin-nginx-fpm#8]
It seems that alias sub server is not supported.

**Comment by Real-Gecko** [comment]
As far as I can remember there are not much options during domain creation:
- Top-level
- Subserver
- Alias
- Alias with own email
  Do you mean creation of subserver for already existing alias server?

**Comment by sinasalek** [comment]
For the last two options it does not work. PHP-FPM does not appear in list of features for alias server

**Comment by Real-Gecko** [comment]
Hmm, as far as I understand alias server is just a domain that has one name, that points to another top level domain, like redirecting. So I think there's no point in adding support for this.

**Comment by sinasalek** [comment]
Yes but it's not a redirect, it's a host that uses the same root folder as its parent. So it needs nginx config all the same

**Comment by Real-Gecko** [comment]
As far as I remember core Nginx module behaves the same way.
Interesting, what's the point of such setup?

**Comment by sinasalek** [comment]
One example is a multi blog engine like wordpress. you can add alias for different users easily like real-gecko.blogsystem.com. 
Another example is having multiple domains for the same system. all control panels including cpanel, directadmin support it with under a different names
Anyway nginx core module and apache module both support it

**Comment by kintaro1981** [comment]
is this supported?
do you suggest to use this module or is better to set sub-server manually?

**Comment by Real-Gecko** [comment]
Nope for now I did not implement this functionality to the module. I'll see what I can do in a few days.
